packages/theme-provider/src/index.ts

```diff
@@ -1,12 +1,17 @@
-import { jsx, useThemeUI, ThemeProvider as CoreProvider } from '@theme-ui/core'
-import { css } from '@theme-ui/css'
+import {
+  jsx,
+  useThemeUI,
+  ThemeProvider as CoreProvider,
+  IntrinsicSxElements,
+} from '@theme-ui/core'
+import { css, Theme } from '@theme-ui/css'
 import { ColorModeProvider } from '@theme-ui/color-modes'
 import { MDXProvider } from '@theme-ui/mdx'
 import { Global } from '@emotion/core'
 
 const BodyStyles = () =>
   jsx(Global, {
-    styles: theme => {
+    styles: (theme: Theme) => {
```

Comment:
This fails in tests, because Global.styles argument is `unknown` and it's not assignable to `theme`.

We'd either need to do
```tsx
<Global<Theme> styles={correctlyTypedTheme => ({})}>
```
or 
```tsx
jsx(Global, {
  styles: emotionTheme => {
    const theme = emotionTheme as Theme
    //...
  },
});
```

Both are in fact assertions. The second one is just explicit. I'll leave it here as a suggestion.

```suggestion
    styles: emotionTheme => {
      const theme = emotionTheme as Theme
```